packages/macro/test/jsx-plural.ts

```diff
@@ -89,6 +89,47 @@ const cases: TestCase[] = [
         }} />;
       `,
   },
+  {
+    input: `
+        import { Trans, Plural, t } from '@lingui/macro';
+        <Plural
+              one={
+                <Trans>
+                  Care Navigator{" "}
+                  <Badge
+                    aria-label={t\`You have unread message\`}
+                  >
```

Comment:
Let's focus in test in one exact case what we are want to check. Make a test case as minimal as possible (without `aria-label={t\`You have unread message\`}`)